### PR TITLE
Adding README file

### PR DESCRIPTION
**Pull request by TzviZeller**
Please review the README file, I have tested most of the steps. Let me know if there are any corrections that need to be made or if there are any missing steps I should add.

### CODE REVIEW

File: 10/README.yaml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        <<: *BaseClientRun
+        arguments:
+        - sh
+        - -c
+        - >-
+          'exec mysql -hsome.mariadb.host -usome-mariadb-user -p'
+    - description: >-
+        You will have to enter the password for `some-mariadb-user` to log on,
+        even though there might not be a prompt to enter password due to
+        limitation of `kubectl run --attach`.
```
Comment (deci-ftw) [inline 10/README.yaml]: limitations of

File: 10/README.yaml [hunk 2]
```diff
@@ -36,10 +36,10 @@ _templates:
 
 overview:
   description: |-
-    This image contains an installation MariaDB 10.x.
+    This image contains of installation MariaDB 10.x.
```
Comment (deci-ftw) [inline 10/README.yaml]: This still needs to be fixed

**Comment by TzviZeller** [comment]
Pipeline verification flagged an issue with version numbers and as such I created another PR to fix this and once that is merged this one can be reviewed.

File: 10/README.md [hunk 3]
```diff
@@ -0,0 +1,618 @@
...
+
+```shell
+kubectl exec -it some-mariadb -- sh -c 'exec mysqldump --all-databases -uroot -p"$MYSQL_ROOT_PASSWORD"' > /some/path/all-databases.sql
+```
+
+If your container was not started with a `MYSQL_ROOT_PASSWORD` value, substitute `"$MYSQL_ROOT_PASSWORD"` with the password of the root user. Alternatively, you can use another pair of username as password for `-u` and `-p` arguments.
+
+# <a name="using-docker"></a>Using Docker
+
+Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
```
Comment (rafalbiegacz) [inline 10/README.md]: Launcher container documentation --> Marketplace container documentation

Comment (rafalbiegacz) [reply 10/README.md]: https://cloud.google.com/launcher/docs/launcher-container --> https://cloud.google.com/marketplace/docs/container-images

File: 10/README.yaml [hunk 4]
```diff
@@ -0,0 +1,252 @@
+#MariaDB# Copyright (C) 2018 Google Inc.
```
Comment (rafalbiegacz) [inline 10/README.yaml]: Do we need this string?

File: 10/README.md (hunk 3 above)
Comment (deci-ftw) [reply 10/README.md]: I think https://cloud.google.com/marketplace/docs/marketplace-container would be the correct replacement for this URL. 

I updated the docgen tool in [PR #716](https://github.com/GoogleCloudPlatform/runtimes-common/pull/716) of the runtimes-common repo to use this URL

File: 10/README.yaml (hunk 4 above)
Comment (TzviZeller) [reply 10/README.yaml]: I have removed it

File: 10/README.md (hunk 3 above)
Comment (TzviZeller) [reply 10/README.md]: Done

**Pull request merged**